coralibre-sdk/sdk/src/main/java/org/coralibre/android/sdk/PPCP.java

```diff
@@ -40,6 +36,8 @@
 	public static void init(Context context) {
 		// TODO: there's no else branch, that's bad.
 		if (ProcessUtil.isMainProcess(context)) {
+			DatabaseAccess.init(context);
+				// TODO check: is this indeed the app context?
```

Comment:
I intentionally add indentations since in my opinion this makes it clearer to which line(s) of code the comments refer. A comment with an extra indentation belongs to the line above.

Reply:
I kind of get your point, but you'd usually put a comment before the line or block it's about and indent it at the same level as a line of code would be indented at that point. Any linter or auto-formatter with a common code style would change your comment's indentation, so please unindent it and move it above the `DatabaseAccess.init(context)` line.